projects/convert pdf to text/converter1.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Sep  9 16:38:39 2020
+
+@author: Admin
+"""
+
+import PyPDF2
+
+pdfobj = open('/path/to/pdf', 'rb') #Provide the path for your pdf here
+
+pdfread = PyPDF2.PdfFileReader(pdfobj)
+
+x = pdfread.numPages
+
+
+for i in range(x):
+    pageObj = pdfread.getPage(i)
+    with open('/path/to/txt', 'a+') as f: #Provide the path for the output text file
```

Comment:
define BASE_DIR and store the output path if user not given any output path store on some location and and return path

Reply:
Ok

Reply:
Hey, I have added the Base directory and the user input for pdf and txt file paths.